file-formats/sush/type/zif_aff_sush_v1.intf.abap

```diff
@@ -0,0 +1,186 @@
+      "! Unmaintained Object Status
+      u_flag type ty_okflag value 'U',
+      "! <p class="shorttext">Authorization Check Inactive</p>
+      "! Authorization Check Inactive
+      n_flag type ty_okflag value 'N',
+    end of co_okflag.
+
+  types:
+    "! <p class="shorttext">Application Name</p>
+    "! Name of Application
+    ty_name type c length 30,
+    "! <p class="shorttext">Application Type</p>
+    "! Type of Application
+    ty_type type c length 2.
+
+  types:
+    "! <p class="shorttext">General</p>
+    "! General
+    "! $required
+    begin of ty_general,
+      "! <p class="shorttext">Application Name</p>
+      "! Application Name of Authorization Default Value.
+      "! $required
+      name   type ty_name,
+      "! <p class="shorttext">Type</p>
+      "! Type of Authorization Default Value.
+      "! $required
+      type   type ty_type,
+      "! <p class="shorttext">Maintenance Mode</p>
+      "! Maintenance Mode.
+      "! $values {@link zif_aff_sush_v1.data:co_usob_status_flag}
+      "! $required
+      okflag type ty_usob_status_flag,
+    end of ty_general .
+
+  types:
+    "! <p class="shorttext">Authorization Value</p>
+    "! Authorization Value
+    ty_val type c length 40.
+
+  types:
+    "! <p class="shorttext">Default Authorization Values</p>
+    "! Default Authorization Values
+    begin of ty_value,
+      "! <p class="shorttext">From</p>
+      "! From Value
+      "! $showAlways
+      low  type ty_val,
+      "! <p class="shorttext">To</p>
+      "! To Values
+      "! $showAlways
+      high type ty_val,
+    end of ty_value .
+
+  types:
+    "! <p class="shorttext">Authorization Fields</p>
+    "! Authorization Fields
+    ty_values  type standard table of ty_value with default key.
+
+  types:
+    "! <p class="shorttext">Authorization Field</p>
+    "! Authorization Field
+    ty_field type c length 10.
+
+  types:
+    "! <p class="shorttext">Authorization Field Details</p>
+    "! Authorization Field Details
+    begin of ty_fldtablestruc,
+      "! <p class="shorttext">Field</p>
+      "! Name of Authorization Field
+      "! $required
+      field  type ty_field,
+      "! <p class="shorttext">Values</p>
+      "! Name of Authorization Field Value
+      values type ty_values,
+    end of ty_fldtablestruc .
+
+  types:
+    "! <p class="shorttext">Authorization Fields</p>
+    "! Authorization Fields
+    ty_authorization_fields  type standard table of ty_fldtablestruc with default key.
+
+  types:
+    "! <p class="shorttext">Authorization Objects Details</p>
+    "! Authorization Objects Details
+    begin of ty_objtablestruc,
+      "! <p class="shorttext">Object</p>
+      "! Name of Authorization Object
+      "! $required
+      object type c length 10,
+      "! <p class="shorttext">Object Description</p>
+      "! Authorization Object Description
+      "! $showAlways
+      ttext  type zif_aff_types_v1=>ty_description_60,
```

Comment:
In the abap file format of SUSH we should not have infos (like descriptions) of other objects.

Currently you cannot display infos of other objects in the ADT Editor. But we will offer the possibility in the future.

Reply:
Agree, if it is not part of the model, we should remove it.